wireapp/outlook-addin
# Kj/wir 21 make add in work for outlook mobile

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

The Wire outlook add-in failed on mobile

### Causes (Optional)

The Wire Outlook add-in failed on mobile because the minimum version requirement for organizer.getAsync is 1.7, but iOS/Android only support up to 1.5.

### Solutions

Distinguish between the platforms and get the name of the organizer through the mailbox.userProfile.displayName on mobile devices.

### Testing

#### How to Test

Open the Outlook Add-in on a PC, Android, and iOS device, and check if the meeting link is parsed into the location field and the invitation message is parsed into the appointment body.

#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
